index-pl.html

```diff
@@ -562,25 +562,25 @@ <h2>How To Stop An Epidemic</h2>
       </span>
     </aside>
 
-    <h2>The Human Cost</h2>
+    <h2>Koszt ludzki</h2>
 
     <p>
-      We can take our understanding one step further by expressing the power of masks in more human terms.
-      Masks save lives by reducing the chance of infection which, in turn, shrinks the extent of the epidemic.
+      Możemy poszerzyć nasze rozumienie krok dalej i wyrazić moc masek za pomocą bardziej przystępnych pojęć.
+      Maski ratują życie poprzez redukowanie szansy zarażenia, co w konsekwencji zmniejsza wymiar epidemii.
```

Comment:
"zmniejsza wymiar epidemii" -> "hamuje postęp epidemii"?

Reply:
o krok dalej

Reply:
> "zmniejsza wymiar epidemii" -> "hamuje postęp epidemii"?
różnica w postępie i rozmiarze to tak jak w pochodnej i funkcji. Konsekwencją zmniejszenia postępu jest zmiana wymiaru.

Reply:
No właśnie, i co teraz. Ja bym raczej zostawił tak jak jest z tym zmniejszaniem wymiaru, ale możecie mnie przegłosować.

Reply:
Proponuję: "rozmiar epidemii"

Reply:
Kompromis, który mogę zaakceptować. Zmieniłem też redukowanie->zmniejszanie